### ISSUE TITLE
FAILED a proof submission for some reason

### ISSUE THREAD

**Issue by ArtoriaShirou** [mel-project/melminter#16]
## Details

During the minting process this error was reported:

```plain
FAILED a proof submission for some reason : could not access network: fatal network error: not enough money for denomination MEL
```

Normally if you get this error, the program will automatically re-mint the ERG, but it doesn't actually re-mint it!

I have actually encountered this error on many machines (especially during the first round of minting), only two of the samples are given here.

Sample 1:

![image](https://user-images.githubusercontent.com/95868829/222504213-ec5a0c81-48fb-4d2a-841f-0733b79c3b82.png)

Sample 2:

![image](https://user-images.githubusercontent.com/95868829/222504251-707f8946-1897-41c5-b6d3-d90b7629957e.png)

**Comment by ArtoriaShirou** [comment]
This has been fixed in `0.8.2`.

**Comment by ArtoriaShirou** [comment]
0.8.2 will still have this bug.
